AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -2324,6 +2324,56 @@ - (void)testShouldChangeDistributionGroupIdIfStoredIdDoesntMatchDownloadedId {
   [utilityMock stopMocking];
 }
 
+- (void)testShouldNotAttemptUpdateIfKeychainIsInaccessible {
+
+  // If
+  id distributeMock = OCMPartialMock(self.sut);
+  id utilityMock = [self mockMSPackageHash];
+  [MSMockKeychainUtil mockStatusCode:errSecInteractionNotAllowed forKey:kMSUpdateTokenKey];
+  [self.settingsMock setObject:kMSTestReleaseHash forKey:kMSDownloadedReleaseHashKey];
+  [self.settingsMock setObject:kMSTestDistributionGroupId forKey:kMSDistributionGroupIdKey];
+  [self.settingsMock setObject:kMSTestDownloadedDistributionGroupId forKey:kMSDownloadedDistributionGroupIdKey];
+
+  // Then
+  OCMReject([distributeMock requestInstallInformationWith:OCMOCK_ANY]);
+  OCMReject([distributeMock checkLatestRelease:OCMOCK_ANY distributionGroupId:OCMOCK_ANY releaseHash:OCMOCK_ANY]);
+
+  // When
+  [distributeMock startUpdate];
+
+  // Stop mocking
+  [distributeMock stopMocking];
+  [utilityMock stopMocking];
+  [MSMockKeychainUtil mockStatusCode:noErr forKey:kMSUpdateTokenKey];
```

Comment:
If you use keychain mock, you must call `stopMocking` to clean it up.

Reply:
It is called in tearDown